cfy-go/scaling.go

```diff
@@ -143,12 +152,11 @@ func scalingGroupsOptions(args, options []string) int {
 			var deployment string
 			var scalegroup string
 			var nodeType string
+
 			operFlagSet.StringVar(&deployment, "deployment", "",
 				"The unique identifier for the deployment")
 			operFlagSet.StringVar(&scalegroup, "scalegroup", "",
 				"The unique identifier for the scalegroup")
-			operFlagSet.StringVar(&nodeType, "node-type",
-				cloudify.KubernetesNode, "Filter by node type")
```

Comment:
Use:
```
operFlagSet.StringVar(&nodeType, "node-type",
				os.Getenv("CFY_K8S_NODE_TYPE"), "Filter by node type")
```

Reply:
@0lvin check https://github.com/cloudify-incubator/cloudify-rest-go-client/tree/testing/cfy-go module it is up to date